src/app/features/edit-nonce-drawer/edit-nonce-drawer.tsx

```diff
@@ -31,10 +32,14 @@ export function EditNonceDrawer() {
   const { errors, setFieldError, setFieldValue, validateField, values } = useFormikContext<
     SendFormValues | TransactionFormValues
   >();
+  const { data: nextNonce } = useNextNonce();
   const [loadedNextNonce, setLoadedNextNonce] = useState<number | string>();
   const navigate = useNavigate();
 
-  useOnMount(() => setLoadedNextNonce(values.nonce));
+  useOnMount(() => {
+    if (!values.nonce) setFieldValue('nonce', nextNonce?.nonce);
```

Comment:
Can you help me understand this? Why would opening the edit nonce drawer affect the nonce value? Couldn't this mean the user opening/closing the edit drawer without making any changes, runs this mount code, potentially change the nonce value?

Reply:
I added this back (it was here before) bc the edit nonce drawer at times launches w/out a form value set. It must be slow to load the nonce at times? This just ensures the nonce form value is set when it opens. It looks like before we used `isUndefined` thou in the check so I can make that change here.